syz-manager/manager.go

```diff
@@ -544,6 +554,11 @@ func (mgr *Manager) runInstance(index int) (*Crash, error) {
 		return nil, fmt.Errorf("failed to setup port forwarding: %v", err)
 	}
 
+	_, err = inst.Copy(mgr.pcsBitmapPath)
```

Comment:
The file will not necessary end in the target at "/bitmap". Some VMs copy it into /, some into /tmp or /data, for some it's configurable by user.
Also "/bitmap" may already exist. I would suggest something like "syz-cover-bitmap".

Reply:
Do you mean executor should get the path of bitmap from manager at run time?
But not just specify a default path for it.